### ISSUE TITLE
Why is android databinding enabled?

### ISSUE THREAD

**Issue by mmueller-abacus** [gcacace/android-signaturepad#98]
Does android-signaturepad need databinding? Because databinding is enabled in gradle file.

**Comment by Wayn0** [comment]
I personally have not looked into it, only found repo a few days ago.
But if you want to build with progaurd you will need.

`-dontwarn android.databinding.**
-keep class android.databinding.** { *; }`

**Comment by scottschmitz** [comment]
Those proguard rules would be fixed by PR #106

**Comment by voghDev** [comment]
Yes it does, if you have a look at `package com.github.gcacace.signaturepad.utils.SignaturePadBindingAdapter`, it uses various `@BindingAdapter` annotations. So I guess it is mandatory to include it.

I wonder if the library can run in an alternative way that gets rid of databinding
